R/tbl_likert.R

```diff
@@ -130,37 +140,198 @@ tbl_likert <- function(data,
   result
 }
 
-
+#' Add column with N to a Likert table
 #' @export
-#' @rdname tbl_likert
-add_n.tbl_likert <- function(x, ...) {
+#' @inheritParams gtsummary::add_n.tbl_summary
+#' @param x Object with class `tbl_likert` from the [tbl_likert()] function
+add_n.tbl_likert <- function(x,
+                             statistic = "{n}",
+                             col_label = "**N**",
+                             footnote = FALSE,
+                             last = FALSE,
+                             ...) {
   # check inputs ---------------------------------------------------------------
   if (!inherits(x, "tbl_likert"))
     stop("`x=` must be class 'tbl_likert'", call. = FALSE)
 
-  # add n column ---------------------------------------------------------------
+  data <- x$inputs$data
+  include <- x$inputs$include
+
+  if ("n" %in% names(x$table_body))
```

Comment:
To make this consistent with the other gtsummary functions, we should update `tbl_likert()` to return the named list of calls, and I think the added N column is called `"stat_n"`?